docs/💖 Free Training/Alchemys Road to Web3.md

```diff
@@ -0,0 +1,18 @@
+# Alchemy's road to Web3 :globe:
```

Comment:
Whoops, wrong emoji name. 😅
```suggestion
# Alchemy's road to Web3 :globe_with_meridians:
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix